ansible_collections/arista/avd/roles/eos_designs/doc/fabric-variables.md

```diff
@@ -14,13 +14,24 @@
   - IBGP (only with OSPF or ISIS variants in underlay)
 - Only summary network addresses need to be defined. IP addresses are then assigned to each node, based on its unique device id.
   - To view IP address allocation and consumption, a summary is provided in the auto-generated fabric documentation in Markdown and CSV format.
-- The variables should be applied to all devices in the fabric.
 
 *Only supported with core_interfaces data model.
 
+## Flagging a Device as Not Deployed
+- It is possible to provision configurations for a complete topology but flag devices as undeployed using the host level variable `is_deployed: false`. 
+- By default, this will have no impact within the `eos_designs` role. Configs will still be generated by the `eos_config_deploy_cli` role. 
```

Comment:
so I doublechecked technically this is `eos_cli_config_gen` which generates the config - whether it is then deployed through CVP or EAPI directly to the devices.
 
the `eos_config_deploy_eapi` (I think I got the name wrong in my previous review) is what I had in mind in the sense that `is_deployed` is ignored by this role it seems (when reading the doc) but then the validate tests would not fail provided either I think with this PR for the deployed hosts.

Reply:
Makes sense, I'll clarify